test/common/http/codec_impl_fuzz_test.cc

```diff
@@ -245,13 +245,14 @@ class HttpStream : public LinkedObject<HttpStream> {
         if (response) {
           auto headers =
               fromSanitizedHeaders<TestResponseHeaderMapImpl>(directional_action.headers());
-          ConnectionManagerUtility::mutateResponseHeaders(headers, &request_.request_headers_,
-                                                          *context_.conn_manager_config_,
-                                                          /*via=*/"", stream_info_, /*node_id=*/"");
-          // Check for validity of response-status explicitly, as encodeHeaders() might throw.
+          // Check for validity of response-status explicitly, as mutateResponseHeaders() and
+          // encodeHeaders() might bug.
```

Comment:
Is there any specific link to where we can see the bug here? Also wondering if this will be part of TODO for the future

Reply:
Well, it's line [518](https://github.com/alyssawilk/envoy/blob/1a506ef6bdb7a26bb761d52b10131312059179e7/source/common/http/utility.cc#L518) that was changed to `ENVOY_BUG()` in #21713 . Does this help? Forgive me. I am German and my English expression could be wrong here.